examples/basics/scene/nested_viewbox.py

```diff
@@ -87,7 +87,7 @@
 vb1.camera.interactive = False
 
 # bottom-left (+y down)
-vb11 = scene.widgets.ViewBox(parent=vb1.scene, name='vb11', 
+vb11 = scene.widgets.ViewBox(parent=vb1.scene, name='vb11', border_width=2e-3, 
```

Comment:
In this case it is because a viewbox is embedded inside another. One of the uses we envisioned for vispy was to design publication-ready figures using physical units. For example, you might want a plot widget that is 4 cm wide with 1 mm border lines, and these sizes do not necessarily correspond to an integral number of pixels.